EpiAware/src/EpiLatentModels/manipulators/ConcatLatentModels.jl

```diff
@@ -110,22 +110,23 @@ Generate latent variables by concatenating multiple latent models.
     @assert all(x -> x > 0, dims) "Non-positive dimensions are not allowed"
     @assert sum(dims)==n "Sum of dimensions must be equal to the dimension of the latent variables"
 
-    @submodel final_latent, latent_aux = _concat_latents(
-        latent_models.models, 1, Real[], [], dims, latent_models.no_models)
+    @submodel final_latent = _concat_latents(
+        latent_models.models, 1, nothing, dims, latent_models.no_models)
```

Comment:
This has an arg as a `Nothing` type but then it varies to another type... this might be a case where `map` works better than recursive function definition. Possible issue.

Reply:
I think this is really another issue to explore? If we see a performance regression that might be an indication of a problem.